common/views/components/PrototypeSearchForm/PrototypeSearchForm.tsx

```diff
@@ -39,6 +39,7 @@ type Props = {
   aggregations: CatalogueAggregations | null;
   isImageSearch: boolean;
   shouldShowFilters: boolean;
+  isCollectionsPage: boolean;
```

Comment:
wondering if this would be better named something like shouldShowSort, makes it more reusable if we  put this component somewhere else and is more explicit

Reply:
better solution agreed.